src/common/CoreMediaIO/DeviceAbstractionLayer/Devices/Sample/Assistant/Server/Device/CMIO_DPA_Sample_Server_VCamDevice.cpp

```diff
@@ -27,7 +38,10 @@ namespace CMIO { namespace DPA { namespace Sample { namespace Server
         Device()
 	{
 		CreateStreams();
-        mFrameSize = 1280 * 720 * 2;
+        
+        obs_video_info ovi;
```

Comment:
Please pull this logic into a function since I think it's a duplicate of what you have above